### ISSUE TITLE
[NPC] Bach'lor - Missing Waypoints

### ISSUE THREAD

**Issue by Rushor** [dalaranwow/dalaran-wow#3832]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:
http://wotlk.openwow.com/npc=18258 Has no waypoints
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
http://wotlk.openwow.com/npc=18258 Has no waypoints
**Expected behaviour**: Tell us what should happen instead
http://wotlk.openwow.com/npc=18258 should have waypoints as you can see on the map of the openwowlink

**Include proofs for this behaviour**
http://wotlk.openwow.com/npc=18258#map
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wotlk.openwow.com/npc=18258
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Comment by Rushor** [comment]
DB/Creature: Add movement and formation for Bach'lor

* commit: 05fc433b454d3237cddf67373754efc0495fee7e

The issue is fixed on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intented on ptr, however i would like to add the combat spell for npc http://wotlk.openwow.com/npc=17130
``` sql
-- Talbuk Stag SAI
SET @ENTRY := 17130;
UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,0,0,0,0,0,100,0,3000,5000,16000,21000,11,32019,0,0,0,0,0,2,0,0,0,0,0,0,0,"Talbuk Stag - In Combat - Cast 'Gore'");
```

**Comment by Rushor** [comment]
DB/Creature: Talbuk Stag - Add combatability 'Gore' in Nagrand 

* updates: https://github.com/dalaranwow/dalaran-wow/issues/3832
* commit: bf73c2899bb4b208fc9a1befae62e3ea05033547

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
